src/app/views/NoteCreate.svelte

```diff
@@ -113,11 +174,19 @@
     thunk.status.subscribe(status => {
       if (
         Object.values(status).length === thunk.request.relays.length &&
-        Object.values(status).every(s => s.status === PublishStatus.Pending)
+        Object.values(status).every((s: any) => s.status === PublishStatus.Pending)
       ) {
         showPublishInfo(thunk)
       }
     })
+    if (emitter) {
+      emitter.on(DVMEvent.Progress, (url: string, event: TrustedEvent) => {
+        $signer.nip04.decrypt(dvmPubkey, event.content).then(data => {
+          data = JSON.parse(data)[0]
+          showInfo(data[2] || "You note is " + data[1])
```

Comment:
Try catch this, and add a `!` at the end of the message (also fix the typo, you -> your)